code/modules/antagonists/protagonists/protagonist.dm

```diff
@@ -0,0 +1,39 @@
+/datum/antagonist/protagonist
+	name = "Protagonist"
+	roundend_category = "protagonists"
+	antagpanel_category = "Protagonists"
+	job_rank = ROLE_PROTAGONIST
+	show_to_ghosts = TRUE
+	///Some protagonists have min ages
+	var/min_age = 16
+	///Some protagonists have max ages
+	var/max_age = 99
+	///Outfit to put onto the character
+	var/outfit_type = /datum/outfit/wizard
+
+
+/datum/antagonist/protagonist/on_gain()
+	. = ..()
+	equip_protagonist()
+	create_objectives()
+
+/datum/antagonist/protagonist/proc/equip_protagonist()
+	if(!owner)
+		CRASH("Antag datum with no owner.")
+	var/mob/living/carbon/human/H = owner.current
```

Comment:
better var names, please

https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names